src/command.js

```diff
@@ -7,7 +7,7 @@ export function processArguments(args, commandName, RedisMock) {
   }
   commandArgs = commandArgs.map(
     // transform non-buffer arguments to strings to simulate real ioredis behavior
-    arg => (arg instanceof Buffer ? arg : arg.toString())
+    arg => (arg instanceof Buffer || arg === null ? arg : arg.toString())
```

Comment:
Should this also handle `undefined`? Unfortunately I don't think checking for a truthy value will work since `0` will resolve to false but would be a valid value.

Reply:
I don't believe `undefined` is a valid value in ioredis, even if it seems like it might be in ioredis-mock.  Can you show me a use case where you might get `undefined` in ioredis?

Reply:
I think `undefined` should be treated as a userland bug in this case. I can't think of a usecase where sending undefined would be intentional?

Reply:
Ok cool. The only reason for suggesting it is that in my testing it was accepted by `ioredis`, storing it as an empty value, so I was thinking of ensuring the functionality matches. However I do agree it's poor usage to pass in `undefined`.

Reply:
ok, that's very interesting. Do you know if this is an intentional behavior on ioredis' part? If so then maybe we should support it. The purpose of `ioredis-mock` is after all to behave exactly like `ioredis`, wherever we can and where it makes sense. If you have to refactor code that works in ioredis in order for it to work in ioredis-mock that's certainly not ideal.

Reply:
My suspicion is it's a product of not planning for someone to attempt to set `undefined`.  But to be fair, I've never tried it. I wouldn't have thought of it.  @stipsan I can update the PR to include `undefined` if you want.  Just let me know.

Reply:
Didn't see your comment @kkragenbrink, I think that as long as ioredis-mock itself does not return `undefined` by mistake I think it's an unlikely scenario. It's more likely that the commands and other features that isn't implemented yet will be problematic in userland.
That said, if anyone makes a PR that allows sending `undefined` I'll merge it for sure 😄